Docker/download_medical_models.py

```
@@ -0,0 +1,203 @@
+"""Discover and optionally download medical GGUF models from Hugging Face.
+
+Compatibility target: llama-cpp-python (GGUF files).
+"""
+
+from __future__ import annotations
+
+import argparse
+import json
+import os
+from pathlib import Path
+from typing import Iterable
```

Ruff (0.15.7)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)